Find a functionally equivalent code:android.graphics.Matrix.preTranslate:COMMENT
Method Modifier: public      
Comment:/**
 * Preconcats the matrix with the specified translation.
 * M' = M * T(dx, dy)
 */

@@ -1,3 +1,4 @@
 {
-    return native_preTranslate(native_instance, dx, dy);
+    native_preTranslate(native_instance, dx, dy);
+    return true;
 }

